src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/DesignerExtendersTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using Moq;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class DesignerExtendersTests : IDisposable
+{
+    private readonly Mock<IExtenderProviderService> _extenderServiceMock;
+    private readonly DesignerExtenders _designerExtenders;
+
+    public DesignerExtendersTests()
+    {
+        _extenderServiceMock = new();
+        _designerExtenders = new(_extenderServiceMock.Object);
+    }
+
+    public void Dispose()
+    {
+        _designerExtenders.Dispose();
+    }
+
+    [Fact]
+    public void Dispose_RemovesExtenderProviders()
+    {
+        IExtenderProvider[] providersField = _designerExtenders.TestAccessor().Dynamic._providers;
```

Comment:
We can simplify this name, and service's too.
```suggestion
        IExtenderProvider[] providers = _designerExtenders.TestAccessor().Dynamic._providers;
```